### PR TITLE
Feature/remove legacy code

### PR DESCRIPTION
**Pull request by vik-vik-ironblocks**

### CODE REVIEW

File: src/errors/index.ts [hunk 1]
```diff
@@ -1,9 +1,7 @@
 import * as apiErrors from './api.errors'
-import * as legacyServerErrors from './legacy-server.errors'
-import * as vennClientErrors from './venn-client.errors'
+import * as legacyServerErrors from './server.errors'
```
Comment (uri-chandler-ironblocks) [inline src/errors/index.ts:2]:
```suggestion
import * as serverErrors from './server.errors'
```

File: src/venn-client/index.ts [hunk 2]
```diff
@@ -1,61 +1,51 @@
 import axios, { AxiosInstance } from 'axios'
-import { isAddress } from 'ethers'
+import { ethers, isAddress } from 'ethers'
 
 import { parseLegacyServerError } from '@/helpers'
```
Comment (uri-chandler-ironblocks) [inline src/venn-client/index.ts:4]:
```suggestion
import { parseServerError } from '@/helpers'
```

**Review by uri-chandler-ironblocks — APPROVED** [review]
💯 

I suggested 2 small changes.
We'll probably refine the README.md a bit more in the next couple of days.

Thanks!

File: src/venn-client/index.ts (hunk 2 above)
Comment (vik-vik-ironblocks) [reply src/venn-client/index.ts:4]: Thanks - good catch. Made the changes.

**Pull request merged**